docs/usage.md

```diff
@@ -1,79 +1,109 @@
-| [Home](https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/develop/README.md) | 
+| [Home](https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/release/1.0.1/README.md) | 
 |--------------------------------------------| 
 
 # Usage
 
+## Playbook Execution Modes
+
+You can execute the GDPR playbooks in *Test Mode* as well as in *Production Mode*
+
+- The playbooks are by default configured to execute in *Test Mode*
+- The *Test Mode* uses a test email ID for all email communication towards DPO, affected individuals, or DPA
+- The *Production Mode* uses actual email IDs of relevant authorities and stakeholders
+- To change execution mode or test email-ID, follow the below procedure:
+    1. Goto to the *Create Data Compliance Record* playbook under the **10 - SP - GDPR Framework** collection and open the *Configuration* step
+
+        ![Execution Mode Configuration](res/execution-mode-configuration.png)
+
+    2. To change of playbooks execution mode from *Test Mode* to *Production*, Modify the value of the **testMode** variable to `false`
+    3. Similarly, to change the test email ID change value of the **testEmail** variable
+
+        ![Execution Mode Configuration Parameters](res/execution-mode-configuration-parametes.png)
+
```

Comment:
@cs-abhishek-shukla  : There is also the mode can be said as "Pre-production mode", where actual emails goes to the DPO email given but in case of the DPA it fetches the actual DPA email and append "_dummy" at the end.

@cs-rohit-khune  : can you please provide the doc input for the same.